components/blitz/src/omero/cmd/graphs/DuplicateI.java

```diff
@@ -530,7 +802,7 @@ public void adjustGraphPolicy(Function<GraphPolicy, GraphPolicy> adjuster) {
 
     @Override
     public int getStepProvidingCompleteResponse() {
-        return dryRun ? 0 : 5;
+        return dryRun ? 0 : 6;
```

Comment:
The significance is that the response is available before all the steps are complete. This isn't actually consumed by anything when we're not doing a dry run but it's conceivable that something may care someday!